cal_coefficients.py

```diff
@@ -6,40 +6,37 @@
 
 """
 import os
-clear = lambda : os.system('cls')
+
+clear = lambda: os.system('cls')
 clear()
-import numpy as np
-import scipy.io as sio
 from receiver_calibration_func import *
 import glob
-import reflection_coefficient as rc
```

Comment:
Needed starting on line 290

Reply:
Good catch, fixed in a4dbf9d4032ab5fe7c899c0395ff057b0e7768a0